src/modelfree/train.py

```diff
@@ -424,7 +457,7 @@ def train(_run, root_dir, exp_name, num_env, rl_algo, learning_rate, log_output_
     multi_venv = maybe_embed_victim(multi_venv, our_idx, scheduler, log_callbacks=log_callbacks)
 
     single_venv = FlattenSingletonVecEnv(multi_venv)
-    single_venv = single_wrappers(single_venv, scheduler, our_idx,
+    single_venv = single_wrappers(single_venv, scheduler, our_idx, lookback_params=lookback_params,
```

Comment:
I don't think this is needed since `single_wrappers` is a Sacred captured function and so will get `lookback_params` anyway.